src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -264,6 +267,9 @@
                             url: url,
                             method: method,
                             type: 'fetch',
+                            updateDuration: function(element) {
+                                element.textContent = (new Date() - stackElement.start) + 'ms';
```

Comment:
instead of using a closure to keep a reference to `stackElement.start` here (which forces you to create the callback in 2 places to make it available for `startAjaxRequest`), you could instead also pass the start time as an argument in the `setInterval` call, as you do for the DOM node (and then, you can define the function only once as a sibling of `startAjaxRequest` and `finishAjaxRequest`, and reuse it for all requests).

This would also solve the re-purposing of the field.